.github/workflows/ci.yml

```diff
@@ -7,6 +7,7 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
I think the goal of this was to ensure draft PRs do not trigger the pipeline, but it doesn't seem the case (see https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/pull/147/files#r1182957292).
I would remove it until the correct solution is found for that (cc @daniel-butler-irl )

Reply:
as per scrum earlier, we need to add this back (apologies)